cmd/confighandler.go

```diff
@@ -222,7 +224,7 @@ func (conf *DeviceConfigHandler) CreateDeviceFromSpec(deviceDef string) {
 
 	// If this is an RTU over TCP device, a default RTU over TCP should already
 	// have been created of the --rtu flag was specified. We'll not re-check this here.
-	manager := conf.ConnectionManager(connSpec, false, 0, "")
+	manager := conf.ConnectionManager(connSpec, false, 0, "", DefaultTimeout)
```

Comment:
Is the DefaultTimeout here consistent?

Reply:
No, thats not consistent. Will be fixed.